.env

```diff
@@ -1 +1 @@
-REACT_APP_JWT_SECRET=<your-jwt-secret>
\ No newline at end of file
+REACT_APP_JWT_SECRET=ecommerceapp
```

Comment:
You should ideally include .env in gitignore file and put all your environment variables on the deployed server (Vercel in your case)